upgrade_tests/test_existance_relations/cli/test_smartvariables.py

```diff
@@ -73,7 +75,7 @@ def test_positive_smart_variables_by_type(pre, post):
     assert existence(pre, post)
 
 
-@dont_run_to_upgrade(['6.1', '6.7'])
+@dont_run_to_upgrade(['6.1', '6.7', '6.8'])
```

Comment:
same as above ^^

Reply:
Same issue was raised by @jyejare, I think, It will help to track the all unsupported version of the respective test component.

Reply:
okay, however this does not have all version it's only have the one version `6.1` , so either should contain 6.1 to 6.6 too or remove 6.1 ?

Reply:
It works for 6.2,6.3,6.4 and 6.5.

Reply:
okay